### DISCUSSION TITLE
i am unable to run forge test -m command

### DISCUSSION THREAD

**Discussion by rupak21** [Cyfrin/foundry-full-course-cu#849] [Q&A]
forge test -m testPriceFeedVersionIsAccurate
error: unexpected argument '-m' found

Usage: forge test [OPTIONS]

For more information, try '--help'.

**Comment by theirrationalone** [comment]
That `-m` is deprecated. Please use `--match-test` Or `--mt` to test your intended test instead.

**Reply by DCVglobalnetwork** [reply]
doesnt working on Window11

**Comment by DCVglobalnetwork** [comment]
what worked for me, proper development environment, set it up, what I mean, check how you create the folder as well as the local server, etc what you create on it, sounds stupid, but the devil is in the details.

**Comment by PatupaiarehePunk** [comment]
I tried the match test but now I am getting "Nothing to Compile" error

Anyone know how to troubleshoot this?

**Reply by PatupaiarehePunk** [reply]
DINBO DINGO DINNER ON A DIME

I was in the wrong REPO!!!!

:0

**Reply by theirrationalone** [reply]
Forge: starving awhile ;`)
